tests/file-magic-keywords/suricata.yaml

```diff
@@ -0,0 +1,15 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      types:
+        - files
+        - stats
```

Comment:
we enable these output types specially but there are no tests corresponding to these.